index.rst

```diff
@@ -1,54 +1,148 @@
+    |            |        | Original   | Computed  | (sec)     |          | hh:mm:ss |  dd:mm:ss |
+    +============+========+============+===========+===========+==========+==========+===========+
+    | 2020-03-12 |   197  |      126.1 |     -10.7 |  2.0      | HD 68450 |  8:11:01 | -37:17:32 |
+    +------------+--------+------------+-----------+-----------+----------+----------+-----------+
+    | 2020-03-12 |   208  |      212.9 |      84.5 |  2.0      | HD 68450 |  8:11:01 | -37:17:32 |
+    +------------+--------+------------+-----------+-----------+----------+----------+-----------+
+    | 2020-03-13 |   130  |      -94.6 |    -198.6 |  5.0      | HD 98993 | 11:23:12 | -36:09:53 |
+    +------------+--------+------------+-----------+-----------+----------+----------+-----------+
+    | 2020-03-13 |   131  |      -94.6 |    -198.6 |  390.0    | HD 98993 | 11:23:12 | -36:09:53 |
+    +------------+--------+------------+-----------+-----------+----------+----------+-----------+
+    | 2020-03-13 |   132  |      -94.6 |    -198.6 |   30.0    | HD 98993 | 11:23:12 | -36:09:53 |
+    +------------+--------+------------+-----------+-----------+----------+----------+-----------+
+
+The equation to compute the sky position angle is as follows:
+
+.. math:: \mathrm{rot\_sky} = \mathrm{PA} - \mathrm{El} - \mathrm{rot\_angle}.
+   :label: math-rotsky-1
+
+If the rotator has the inverse orientation, the corrected angle should be computed as:
+
+.. math:: \mathrm{rot\_sky} = \mathrm{PA} - \mathrm{El} + \mathrm{rot\_angle}
+   :label: math-rotsky-2
+
+where ``rot_sky`` is the angle eastward of north between camera y-axis and north, ``PA`` is the Parallactic Angle, ``El`` is the target elevation in the local coordinate system and ``rot_angle`` is the Rotator/Nasmyth angle.
+
+.. _section-HD98993:
+
+HD 98993
+--------
+
+This is probably the best dataset to show that the rotator is moving in the wrong direction.
+We have a short discovery exposure (``visitId = 2020031300130``), followed by a long 390s exposure (``visitId = 2020031300130``) and a subsequent 30s exposure (``visitId = 2020031300132``).
+The :ref:`long exposure <figure-2020031300131>` clearly shows arc-like star trails, which is consistent with rotator drift.
+The size of the arcs can be easily estimated since we know the plate scale of the images and, be compared to the expected arc-size in case the rotator is rotating in the wrong direction.
+Visits ``2020031300130`` and ``2020031300132``, although do not show any sign of image streak (due to the shorter exposure times) can be used to validate that the streaks in ``2020031300131`` are continuous and not jitter during the exposure.
+
+In :numref:`figure-2020031300131` we show visit ``2020031300131`` with rulers showing the arc length and its distance to the apparent center of rotation.
+The measurement values are :math:`0.003\deg` and :math:`0.062\deg`, which translates to an angle of :math:`2.77\deg`.
```

Comment:
\deg isn't rendering properly anywhere here... it's just writing "deg" in italics...